### DISCUSSION TITLE
Setting growth media

### DISCUSSION THREAD

**Discussion by sophiez8** [micom-dev/micom#88] [Q&A] [answered]
Dear Micom team,

I have a question regarding the media specification. I want to simulate two species co-cultured in a pre-defined medium, for which I want to bring together the sbml models of the two organisms in MICOM. However, those two models have different names for the external exchange reactions. Would I have to manually map the reaction IDs in order to specify the media or can I just specify the media according to the exchange names of both models subsequently? Or is there a easier way?  

thank you! 

Sophie

**Comment by cdiener** [comment, marked as answer]
To combine the two models in MICOM all exchanged metabolites that describe the same compound in booth models need to have the same ID. When you combine them with MICOM it will then create a single media exchange reaction for that metabolite which you can reference on the growth medium.

**Reply by sophiez8** [reply]
thank you!
